src/utils.js

```diff
@@ -524,4 +535,66 @@ export const getObjectDiff = (obj1, obj2) => {
     return result;
   }, {});
   return difference
+}
+
+export const parseDirectoryFile = (folderContent) => {
+  const decodedContent = yaml.safeLoad(Base64.decode(folderContent))
+  const collectionKey = Object.keys(decodedContent.collections)[0]
+  return decodedContent.collections[collectionKey].order
+}
+
+export const convertFolderOrderToArray = (folderOrder) => {
```

Comment:
<img width="1792" alt="Screenshot 2021-02-23 at 5 37 15 PM" src="https://user-images.githubusercontent.com/39231249/108825280-c8255e80-75fd-11eb-81b9-692e6bfb6805.png">
Didn't investigate it in detail, but seems like this function isn't working as expected, see ^

Reply:
Files belonging to a subfolder are not properly added to the children array, children arrays currently have a length of 1.

Reply:
fixed with [9faab87](https://github.com/isomerpages/isomercms-frontend/pull/350/commits/9faab87c8c2f994077d087476964742359a16b83)